Podfile

```diff
@@ -0,0 +1,7 @@
+platform :ios, '9.0'
```

Comment:
@petrucci34 We could change this to `8.0` to reflect the lowest supported version by the SDK